src/main/java/dev/pavatus/planet/mixin/gravity/LivingEntityMixin.java

```diff
@@ -63,7 +65,16 @@ public LivingEntityMixin(EntityType<?> type, World world) {
             return;
         }
 
-        if (!planet.hasOxygen() && !Planet.hasOxygenInTank(entity)) {
+        if (TardisDimension.isTardisDimension(entity.getWorld())) {
+            if (!TardisDimension.get(entity.getWorld()).get().subsystems().lifeSupport().isEnabled()) {
+                entity.addStatusEffect(new StatusEffectInstance(StatusEffects.WITHER, 1,
```

Comment:
maybe this should be poison instead so the players dont insta die while setting up their tardis

Reply:
not resolved is still wither

Reply:
didnt read dms lol